tests/compiler/std/math.ts

```diff
@@ -1538,7 +1538,7 @@ assert(test_expm1f(NaN, NaN, 0.0, 0));
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 // Math.exp2
 ////////////////////////////////////////////////////////////////////////////////////////////////////
-
+/*
```

Comment:
With exp2 removed, should this be removed as well?

Reply:
I think leave it. May be in future I implement it again and this tests will be necessary

Reply:
ok :)